Disable selinux and systemd support since they check for host headers.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura iodine-0.7.0.orig/src/osflags iodine-0.7.0/src/osflags
--- iodine-0.7.0.orig/src/osflags	2015-02-20 14:59:00.799958145 -0300
+++ iodine-0.7.0/src/osflags	2015-02-20 15:05:54.791135141 -0300
@@ -18,8 +18,8 @@
 		;;
 		Linux)
 			FLAGS="";
-			[ -e /usr/include/selinux/selinux.h ] && FLAGS="$FLAGS -lselinux";
-			[ -e /usr/include/systemd/sd-daemon.h ] && FLAGS="$FLAGS -lsystemd-daemon";
+			#[ -e /usr/include/selinux/selinux.h ] && FLAGS="$FLAGS -lselinux";
+			#[ -e /usr/include/systemd/sd-daemon.h ] && FLAGS="$FLAGS -lsystemd-daemon";
 			echo $FLAGS;
 		;;
 	esac
@@ -34,8 +34,8 @@
 		;;
 		Linux)
 			FLAGS="-D_GNU_SOURCE"
-			[ -e /usr/include/selinux/selinux.h ] && FLAGS="$FLAGS -DHAVE_SETCON";
-			[ -e /usr/include/systemd/sd-daemon.h ] && FLAGS="$FLAGS -DHAVE_SYSTEMD";
+			#[ -e /usr/include/selinux/selinux.h ] && FLAGS="$FLAGS -DHAVE_SETCON";
+			#[ -e /usr/include/systemd/sd-daemon.h ] && FLAGS="$FLAGS -DHAVE_SYSTEMD";
 			echo $FLAGS;
 		;;
 	esac
